composer.json

```diff
@@ -21,6 +21,15 @@
         "drupal/console": "~0.10",
         "drupal/adminimal_theme": "^8.1"
     },
+    "require-dev": {
+        "behat/behat": "^3.0",
+        "drupal/drupal-extension": "^3.0",
+        "lakion/mink-debug-extension": "^1.0",
+        "jakub-onderka/php-parallel-lint": "^0.9.2",
+        "mikey179/vfsStream": "~1.2",
```

Comment:
Is it because of the "image scale" service that you already include vfsStream as a dependency now, @kasperg ? :relaxed: